2nd-gen/instrument-with-opentelemetry/functions/.env

```diff
@@ -0,0 +1 @@
+NODE_OPTIONS=--require=./tracing.js
```

Comment:
```suggestion
# This loads the telemetry library before functions framework because
# The Open Telemetry SDK must be the first thing that runs
NODE_OPTIONS=--require=./tracing.js
```